### DISCUSSION TITLE
How to meet the SLA recommendation?

### DISCUSSION THREAD

**Discussion by dsfrederic-cgk** [Azure/azqr#252] [Q&A] [answered]
I'm looking for the documentation how I can meet the SLA recommendations. Can anyone guide me to the correct documentation?

**Comment by cmendible** [comment, marked as answer]
SLAs shown by `azqr` (In the Inventory sheet) are calculated based on the service instance configuration. It's provided so you don't have to go and read the docs to get the SLA you should expect for a given service.

For some services such as AKS or APIM you'll also get a recommendation (In the ImpactedResources sheet) if the services is configured with the Free or Developer tier cause no SLA will be provided by Azure and that is a risk for your production workloads.
